screenpipe-server/src/video.rs

```diff
@@ -355,21 +355,8 @@ async fn start_ffmpeg_process(output_file: &str, fps: f64) -> Result<Child, anyh
         "pad=width=ceil(iw/2)*2:height=ceil(ih/2)*2",
     ];
 
-    if env::consts::OS == "windows" {
-        // TODO switch back to libx264 when ffmpeg is updated in pre_build.js
-        // Use MPEG-4 encoder for Windows
-        args.extend_from_slice(&[
-            "-vcodec",
-            "mpeg4",
-            "-q:v",
-            "5", // Adjust quality (1-31, lower is better)
-            "-preset",
-            "ultrafast",
-        ]);
-    } else {
-        // Use libx264 for other platforms
-        args.extend_from_slice(&["-vcodec", "libx264", "-preset", "ultrafast", "-crf", "23"]);
-    }
+    // Use libx264 for all platforms
+    args.extend_from_slice(&["-vcodec", "libx264", "-preset", "ultrafast", "-crf", "23", ]);
```

Comment:
how did you install ffmpeg? @tribhuwan-kumar

Reply:
[ffmpeg](https://winstall.app/apps/Gyan.FFmpeg) is avaiable in window's official pacakge manager `winget` also for development purpose I installed ffmpeg from vcpkg.

Reply:
so this will crash

you should use the ffmpeg version from the `pre_build.js` which is packaged in the release of the app (or change this to use a "better" version of ffmpeg) the if i did is because the ffmpeg does not support the arg given here

Reply:
#194 would probably resolve all the problems i guess

Reply:
> so this will crash
> 
> you should use the ffmpeg version from the `pre_build.js` which is packaged in the release of the app (or change this to use a "better" version of ffmpeg) the if i did is because the ffmpeg does not support the arg given here

Okay, I'm uninstalling the downloaded version of ffmpeg from winget & vcpkg and try to rebuild it with [`pre_build.js`](https://unlimited.dl.sourceforge.net/project/avbuild/windows-desktop/ffmpeg-7.0-windows-desktop-vs2022-default.7z?viasf=1) version of ffmpeg.

Reply:
>you should use the ffmpeg version from the `pre_build.js`

I did some digging about the `avbuild/windows-desktop/ffmpeg-7.0-windows-desktop` and find out it doesn't supports the `libx264` encoder:
<img src="https://github.com/user-attachments/assets/4044ab60-8935-4a80-a105-5fe4d909e067" width="300px" />

 so I test with multiple encoder and decided to use `h264_mf` (Microsoft Media Foundation) which is supported by avbuild version of ffmpeg & its working:

 <img src="https://github.com/user-attachments/assets/6277dbc5-8e56-4945-973d-afec92a1c413" width="300px" />
 
 Let me know if any problem occurs :)